src/NHibernate.Test/Async/Tools/hbm2ddl/SchemaValidator/SchemaValidateFixture.cs

```diff
@@ -53,34 +53,6 @@ public async Task ShouldVerifySameTableAsync()
 		}
 
 #if NETFX
-		[Test, SetCulture("tr-TR"), SetUICulture("tr-TR")]
-		public async Task ShouldVerifySameTableTurkishAsync()
```

Comment:
Could you please clarify why async code is not generated here? Is it some limitation of Roslyn  parser - so disabled code is not visible?

Reply:
By changing the `targetFramework` to `netcoreapp2.0`, the parser treats directives like `NETFX` as disabled and no symbols and syntax nodes are produced for members inside those directives, which means that code is available only as a plain text.

Reply:
So is it required for parser to have compilable code to produce such symbols/syntax nodes? I mean if for instance we supply NETFX directive under netcoreapp2.0 -  parser still won't be able to process it due to compilation errors?

Reply:
> So is it required for parser to have compilable code to produce such symbols/syntax nodes?

Yes, the generator uses symbols to find where the method is referenced, if the containing type has any async counterparts, if the method has any overrides, if implements any interfaces... Without all this information is not possible to determine what should be converted and what not.

> I mean if for instance we supply NETFX directive under netcoreapp2.0 - parser still won't be able to process it due to compilation errors?

In case when a type or member exists only in NETFX, the parser will create a syntax node for the type/member (as its a valid C# code), but it won't contain the information about the symbol, if target framework would be set to `netcoreapp2.0`, which mean that the code will be partially evaluated.
For this specific test, where the code is valid in both NETFX and NETCore, I've added a special attribute `NetFxOnlyAttribute` which will run only in NETFX, but the code be compiled in both frameworks, which will allow the generator to generate the async code when possible.